lib/pixiv/client.rb

```diff
@@ -54,9 +57,9 @@ def login(pixiv_id, password)
 
     # @param [Integer] member_id
     # @return [Pixiv::Member] member bound to +self+
-    def member(member_id = member_id)
-      attrs = {member_id: member_id}
-      member = Member.lazy_new(attrs) { agent.get(Member.url(member_id)) }
+    def member(mid = member_id)
```

Comment:
I think some people will confuse in this variable name. I suggest to use original variable name the`member_id`.